src/systems/tracked-controls.js

```diff
@@ -16,15 +18,21 @@ module.exports.System = registerSystem('tracked-controls', {
     });
   },
 
-  tick: function () {
-    var gamepads = navigator.getGamepads && navigator.getGamepads();
-    var gamepad;
+  rebuildControllerList: function () {
     var controllers = this.controllers = [];
-    var i;
-    if (!gamepads) { return; }
-    for (i = 0; i < gamepads.length; ++i) {
-      gamepad = gamepads[i];
+    var gamepads = trackedControlsUtils.getGamepadsByPrefix('');
+    for (var i = 0; i < gamepads.length; i++) {
+      var gamepad = gamepads[i];
       if (gamepad && gamepad.pose) { controllers.push(gamepad); }
     }
+  },
+
+  tick: function () {
+    var now = Date.now();
```

Comment:
You don't need to do `Date.now();`. The tick method already gives you the time.
````javascript
tick: function(time) { ... }
````